node/core/approval-voting/src/criteria.rs

```diff
@@ -128,13 +208,13 @@ fn relay_vrf_delay_tranche(
 }
 
 fn assigned_core_transcript(core_index: CoreIndex) -> Transcript {
-	let mut t = Transcript::new(approval_types::ASSIGNED_CORE_CONTEXT);
+	let mut t = Transcript::new(approval_types::v1::ASSIGNED_CORE_CONTEXT);
 	core_index.0.using_encoded(|s| t.append_message(b"core", s));
 	t
 }
```

Comment:
I'd expect this to commit to the claimed list of cores, so it gets hashed in with the AssignmentCert below in https://github.com/paritytech/polkadot/pull/6782/files#diff-bf332a1521acdb786c4a41b8d2ed15930c6675866a4a8bb6493cbd6105af1008L511

Reply:
Now it does!